pkgs/development/libraries/libgit2/default.nix

```diff
@@ -44,11 +45,14 @@ stdenv.mkDerivation (finalAttrs: {
     "-DDLLTOOL=${stdenv.cc.bintools.targetPrefix}dlltool"
     # For ws2_32, refered to by a `*.pc` file
     "-DCMAKE_LIBRARY_PATH=${stdenv.cc.libc}/lib"
+  ] ++ lib.optionals stdenv.hostPlatform.isOpenBSD [
+    # openbsd headers fail with default c90
+    "-DCMAKE_C_STANDARD=99"
   ];
 
   nativeBuildInputs = [ cmake python3 pkg-config ];
 
-  buildInputs = [ zlib libssh2 openssl pcre2 llhttp ]
+  buildInputs = [ zlib libssh2 openssl pcre2 llhttp http-parser ]
```

Comment:
For USE_HTTP_PARSER we need either llhttp or http-parser. I would drop llhttp

Reply:
`http-parser` is deprecated and unmaintained, so if it’s a choice between that and `llhttp`, we should choose the latter.

Reply:
I’ve fixed the CMake parameter and dropped `http-parser`.

Reply:
Messed that up 😅 thanks for taking care for that